src/main/kotlin/pl/put/smartgarden/infra/api/DeviceController.kt

```diff
@@ -43,7 +45,11 @@ class DeviceController(
         ApiResponse(code = 400, message = "Bad request")
     ])
     @ResponseStatus(HttpStatus.OK)
-    fun createMeasures(@RequestBody deviceMeasuresDto: DeviceMeasuresDto): List<MeasureResponse> = deviceFacade.createMeasures(deviceMeasuresDto)
+    fun createMeasures(
+        @RequestBody deviceMeasures: List<MeasureDto>,
```

Comment:
createMeasures przyjmuje parametr {id} w path, skoro i tak go nie wykorzystujemy, a id pobieramy z tokena, to trzeba przemyśleć czy nie zmienić kontrolera z **/devices/{id}/[operacja]** na np. **/device/[operacja]**